api/osb/osb_controller.go

```diff
@@ -178,13 +177,13 @@ func buildProxy(targetBrokerURL *url.URL, logger *logrus.Entry, broker *types.Se
 		logger.Infof("Forwarded OSB request to service broker %s at %s", broker.Name, request.URL)
 	}
 
-	tlsConfig, err := broker.GetTlsConfig()
+	tlsConfig, err := broker.GetTLSConfig()
 
 	if err != nil {
 		return nil, &util.HTTPError{
-			ErrorType:   "BadRequest",
+			ErrorType:   "InternalError",
 			Description: "unable to get tls configuration: " + err.Error(),
-			StatusCode:  http.StatusBadRequest,
+			StatusCode:  http.StatusInternalServerError,
```

Comment:
just return a normal error and the error handler will return status 500, see https://github.com/Peripli/service-manager/blob/master/pkg/util/errors.go#L77
same thing below